pkg/kubelet/prober/prober_manager.go

```diff
@@ -48,6 +48,21 @@ var ProberResults = metrics.NewCounterVec(
 		"pod_uid"},
 )
 
+// ProberDuration stores the duration of a successful probe lifecycle by result as prometheus metrics.
+var ProberDuration = metrics.NewHistogramVec(
+	&metrics.HistogramOpts{
+		Subsystem:      "prober",
+		Name:           "probe_duration_seconds",
+		Help:           "Duration in seconds for a probe response",
+		StabilityLevel: metrics.ALPHA,
+	},
+	[]string{"probe_type",
+		"container",
+		"pod",
+		"namespace",
+		"pod_uid"},
```

Comment:
`pod_uid` isn't meaningful for the use-case of this metric since we already have the pod name. As such I would be in favor of removing it.